FUN.md

```diff
@@ -60,4 +60,45 @@ Verified OK
 
 Store the signature in the Transparency Log and don't store the keys anywhere.
 
-**COMING SOON**
+```
```

Comment:
Do you want to adjust Hard Mode above to say that the public key is also stored in transparency log?